manifests/1.14/deploy/vsphere-csi-node-ds.yaml

```diff
@@ -38,22 +40,30 @@ spec:
             - name: registration-dir
               mountPath: /registration
         - name: vsphere-csi-node
-          image: gcr.io/cloud-provider-vsphere/csi/release/driver:v0.2.1
+          image: cloudnativestorage/vsphere-block-csi-driver:latest
+          imagePullPolicy: "Always"
           env:
             - name: CSI_ENDPOINT
               value: unix:///csi/csi.sock
             - name: X_CSI_MODE
               value: "node"
             - name: X_CSI_SPEC_REQ_VALIDATION
               value: "false"
-#           - name: X_CSI_DEBUG
-#             value: "true"
+            - name: X_CSI_VSPHERE_CLOUD_CONFIG
```

Comment:
Yea, I that would not be a very good situation to be in. You have the potential for security-related issues. Originally creds were only exposed to just the master node which you would argue is held to a higher standard since you're talking control plane at that point.

Reply:
@codenrhoden @dvonthenen 

mounting  `vsphere-config-volume` config-map on vsphere-csi-node pods is optional.

node plugin only needs access to the `csi-vsphere.conf`, when zone feature is enabled, so it can obtain and return accessibleTopology of the node from `NodeGetInfo`.

If we do not have zone/region configured on the cluster then we do not need to mount this credential file on `vsphere-csi-node` pods.